xml/smtp_banners.xml

```diff
@@ -1647,4 +1648,93 @@ The system or service fingerprint with the highest certainty overwrites the othe
     <param pos="1" name="host.name"/>
   </fingerprint>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+        <fingerprint pattern=”^(\w+.\w+.\w)+ Service not available$>
+        <description>match to Service not available error
+        <example service.version=”4.3.2”> 4.3.2 Service not available <example>
+        <param pos="1" name="service.version"/>
+        <param pos=”0” name=”service.vendor” value=”new”/>
+        <param pos=”0” name=”service.family” value=”version”/>
+        <param pos=”0” name=”service.product” value=”version”/>
+        <fingerprint>
+         <fingerprint pattern="^(.+.{4})-\w.+echosting+[^ ]+(cafe+.+.+.[a-z]+[a-z]+[a-z])+ ESMTP$">
+        <description>match to echosting or ESMTP
+        <example service.version="e0000" website.version="cafe24.com"> e0000-e.echosting.cafe24.com ESMTP <example>
+        <param pos="1" name="service.version"/>
+        <param pos="0" name="service.vendor" value="new"/>
+        <fingerprint>
+         <fingerprint pattern="^([a-z]+[a-z]+[a-z]+.)+.cloud.netart +ESMTP +ready$">
+        <description>match to cloud.netart and ESMTP and ready state
+        <example service.version="lbc4"/> lbc4.cloud.netart ESMTP ready <example>
+        <param pos="1" name="service.version"/>
+        <param pos="0" name="service.vendor" value="new"/>
+        <param pos="0" name="service.family" value="version"/>
+        <fingerprint>
+         <fingerprint pattern="^([A-Z]+[A-Z]+[A-Z]+[A-Z])+[^ ]+(\d+[^ ]+\d) (\d\d\d) +Unauthorized$">
+        <description> match to HTTP error
+        <example service.version="HTTP" service.number="1.0" error.number="403"> HTTP/1.0 403 Unauthorized <example>
+        <param pos="1" name="service.version"/>
+        <param pos="2" name="service.number"/>
+        <param pos="3" name="error.number"/>
+        <fingerprint>
+         <fingerprint pattern="^([a-z]+[a-z].[a-z]+[a-z]).([a-z]+[a-z]+[a-z]+\d+\d+\d+\d) ESMTP service ready$">
+        <description> match to ESMTP service numbers
+        <example service.version="hm.sm" service.number="srv1118"/> hm.sm.srv1120 ESMTP service ready <example>
+        <param pos="1" name="service.version" />
+        <param pos="2" name="service.number" />
+        <param pos="0" name="service.family" value="server"/>
+        <fingerprint>
+         <fingerprint pattern="^(\D{2}+\d{3}).secure.ne.jp+ ESMTP">
+        <description> match to secure.ne.jp in Japan
+        <example service.version="aa227"/> aa227.secure.ne.jp ESMTP <example>
+        <param pos="1" name="service.version" />
+        <param pos="0" name="service.vendor" value="japan" />
+        <param pos="0" name="service.family" value="secure"/>
+        <fingerprint>
+         <fingerprint pattern="^(\D+\D+\d+\d).([a-z]+[a-z]+[a-z]+[a-z]+[a-z])+.jp+ ESMTP$">
+        <description> match to domain controller
+        <example service.version="dc51"/> dc51.etius.jp ESMTP <example>
```

Comment:
Be careful about the assertions and assumptions that are expressed as part of a fingerprint.  For example, are you sure that this is a domain controller and that `dc` doesn't stand for Data Center instead?  The `service.vendor` and `service.family` values below are incorrect as well unless there is a vendor named `Japan` with a product family named `etius`.